src/cmd/projects.js

```diff
@@ -0,0 +1,81 @@
+
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import promisify from 'es6-promisify';
+
+const particle = 'particle';
+const libraries = 'libraries';
+const projects = 'projects';
+const community = 'community';
+const mine = 'mine';
+
+const fsMkdir = promisify(fs.mkdir);
+const fsExists = promisify(fs.exists);
+
+function mkdir(name) {
+	return fsMkdir(name)
+		.then(() => true)
+		.catch(error => {
+			if (error.code==='EEXIST')
+				return false;
+			throw error;
+		});
+}
+
+function mkdirp(name) {
+	return fsExists(name).then(exists => {
+		if (!exists) {
+			const parent = path.dirname(name);
+			const makeit = mkdir(name);
+			let promise = makeit;
+			if (parent && parent!=='/') {
+				promise = mkdirp(parent).then(() => makeit);
+			}
+			return promise;
+		}
+	});
+}
+
+
+export class Projects {
+
+	userHomeFolder() {
+		return os.homedir();
```

Comment:
I was assuming Babel added the necessary polyfill along with all the other newer node functions. Does it not do that?